demo/package.json

```
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "build": "webpack",
```

The build script in project-demo-v1 looks like [this](https://github.com/HubSpot/calling-extensions-sdk/blob/project-demo-v1/demo/package.json#L7). Perhaps we could set the `NODE_OPTIONS` here?
```suggestion
    "build": "NODE_OPTIONS=--openssl-legacy-provider webpack",
```

---

Setting the NODE_OPTIONS this way is unfortunately not cross-platform friendly, which is why I had to create the start script and use [cross-env](https://www.npmjs.com/package/cross-env). I also prefer to let the user know that we are using the legacy openssl provider if they use the latest node versions. Ideally, we update the node dependencies in the demo's package.json such that there is no need to use the legacy provider at all, though.

---

Would you have a script for the `build` command to include NODE_OPTIONS=--openssl-legacy-provider?